west-zephyr.yml

```diff
@@ -14,6 +14,8 @@ manifest:
           - net-tools
           - segger
           - tinycrypt
+          - hal_nxp
+          - lvgl
```

Comment:
Please keep the list in alphabetic order. Why `lvgl`?

Reply:
Previously the default config for this board threw errors when lvgl wasn't present:

```
warning: LV_Z_HOR_RES_MAX (defined at boards/arm/mimxrt1060_evk/Kconfig.defconfig:65) defined without a type                                                                  
                                                                                                                                                                              
warning: LV_Z_VER_RES_MAX (defined at boards/arm/mimxrt1060_evk/Kconfig.defconfig:68) defined without a type                                                                  
Parsing /home/mike/golioth-zephyr-workspace/zephyr/Kconfig                                                                                                                    
Loaded configuration '/home/mike/golioth-zephyr-workspace/zephyr/boards/arm/mimxrt1060_evk/mimxrt1060_evk_defconfig'                                                          
Merged configuration '/home/mike/golioth-zephyr-workspace/zephyr/samples/basic/minimal/prj.conf'                                                                              
                                                                                                                                                                              
error: Aborting due to Kconfig warnings  
```

But it appears to be fixed now that our snapshot includes this commit so I'll remove this dependency: https://github.com/zephyrproject-rtos/zephyr/commit/8eead500717e733abab6ac3a0952cd60bea5f8ac